modules/oe_content_publication/config/install/field.field.node.oe_publication.body.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.body
+    - node.type.oe_publication
+  module:
+    - text
+id: node.oe_publication.body
+field_name: body
+entity_type: node
+bundle: oe_publication
+label: Body
+description: ''
+required: false
+translatable: false
```

Comment:
Should be translatable. no? in ticket description.

Reply:
This is fixed.